docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+| dcterms:hasVersion            | URI                | N/A   | A String Literal that represents constituent parts of a resource.                          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:collection                | String Literal     | N/A   | A String Literal that represents the physical archival collection the resource belongs to. |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:isPartOf                  | URI                | N/A   | A URI that represents the physical archival collection a resource belongs to.              |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:memberOfArchivalSeries | URI/String Literal | N/A   | A URI or String Literal that indicates the archival series a resource belongs to.          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:sheetMusic_hostItem    | URI/String Literal | N/A   | A URI or String Literal that indicates the host item for the resource being described.     |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| relators:[term]               | URI/String Literal | N/A   | Use with a role from MARC Code List of Relators role terms.                                |
+|                               |                    |       | Value is either text or URI from a controlled vocabulary (like                             |
+|                               |                    |       | Library of Congress Name Authority File).                                                  |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+
+relatedItem as simple structural metadata
+-----------------------------------------
+Use Case
+^^^^^^^^
+`relatedItem`, with and without attributes, is used in a number of collections to express structural relationships. These relationships will be handled/expressed by default behavior in our next-gen DAMS. In the case of `relatedItem/abstract`, the values present should be handled at the collection level.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[not(@*)]` OR
+:code:`relatedItem[@type='host'][@displayLabel[matches(., 'project') or matches(., 'Project') or matches(., 'Digital Collection') or matches(., 'Project Part') or matches(., 'Is Part Of')]` OR
+:code:`relatedItem/abstract`
+
+Decision
+^^^^^^^^
+Do not migrate.
+
```

Comment:
Even though we're not migrating, can we include some XML as an example so it's clear what information is present?